index.html

```diff
@@ -0,0 +1,222 @@
+</section>
+
+<section class="main-section" id="Variable_scope">
+  <header>Variable scope</header>
+    <p>When you declare a variable outside of any function, it is called a global variable, because it is available to any other code in the current document. When you declare a variable within a function, it is called a local variable, because it is available only within that function.</p>
+    <p>JavaScript before ECMAScript 2015 does not have block statement scope; rather, a variable declared within a block is local to the function (or global scope) that the block resides within. For example the following code will log 5, because the scope of x is the function (or global context) within which x is declared, not the block, which in this case is an if statement.</p>
+    <code>if (true) { var x = 5; } console.log(x); // 5</code>
+    <p>This behavior changes, when using the let declaration introduced in ECMAScript 2015.</p>
+    <code>if (true) { let y = 5; } console.log(y); // ReferenceError: y is
+    not defined</code>
+</section>
+
+<section class="main-section" id="Global_variables">
+  <header>Global variables</header>
+  <p>Global variables are in fact properties of the global object. In web pages the global object is window, so you can set and access global variables using the window.variable syntax.</p>
+  <p>Consequently, you can access global variables declared in one window or frame from another window or frame by specifying the window or frame name. For example, if a variable called phoneNumber is declared in a document, you can refer to this variable from an iframe as parent.phoneNumber.</p>
+</section>
+
+<section class="main-section" id="Constants">
+  <header>Constants</header>
+    <p>You can create a read-only, named constant with the const keyword. The syntax of a constant identifier is the same as for a variable identifier: it must start with a letter, underscore or dollar sign and can contain alphabetic, numeric, or underscore characters.</p>
+    <code>const PI = 3.14;</code>
+    <p>A constant cannot change value through assignment or be re-declared while the script is running. It has to be initialized to a value.</p>
+    <p>The scope rules for constants are the same as those for let block scope variables. If the const keyword is omitted, the identifier is assumed to represent a variable.</p>
+    <p>You cannot declare a constant with the same name as a function or variable in the same scope. For example:</p>
+    <code>// THIS WILL CAUSE AN ERROR function f() {}; const f = 5; // THIS
+      WILL CAUSE AN ERROR ALSO function f() { const g = 5; var g;
+    //statements }</code>
+    <p>However, object attributes are not protected, so the following statement is executed without problems.</p>
+    <code>const MY_OBJECT = {"key": "value"}; MY_OBJECT.key =
+    "otherValue";</code>
+</section>
+
+<section class="main-section" id="Data_types">
+  <header>Data types</header>
```

Comment:
Replace your `header` element with heading element(h1, h2, h3, h4, h5, h6)

Reply:
- [ ] Kindly always include an explanation, or a "why", for each requested change, this will make your comments more useful and clear. eg why using the header as a substitute for your ```<hx>``` is not a good thing